src/pages/dashboard/index.js

```diff
@@ -23,6 +24,15 @@ const Dashboard = () => {
     setSearchVal(e.target.value);
   };
 
+  const onSubmit = (e) => {
+    e && e.preventDefault();
+    try {
+      getUserByName(searchVal).then(setUsers);
+    } catch (e) {
+      throw new Error(`no users named ${searchVal} were found`);
```

Comment:
Let's capture the error here and do `console.error('error getting user by name', e)`